tortoise/verifying_tortoise.go

```diff
@@ -291,243 +371,233 @@ func RecoverVerifyingTortoise(mdb retriever) (interface{}, error) {
 	// and add the corresponding vector (multiplied by the block weight) to our own vote-totals vector.
-	//We then add the vote difference vector and the explicit vote vector to our vote-totals vector.
-
-	t.logger.With().Debug("Processing block", block.Fields()...)
-	t.logger.With().Debug("Getting baseblock", block.BaseBlock)
+	// We then add the vote difference vector and the explicit vote vector to our vote-totals vector.
+	t.logger.With().Debug("processing block", block.Fields()...)
+	t.logger.With().Debug("getting baseblock", block.BaseBlock)
 
 	base, err := t.bdp.GetBlock(block.BaseBlock)
 	if err != nil {
 		return fmt.Errorf("can't find baseblock")
 	}
 
 	t.logger.With().Debug("block supports", types.BlockIdsField(block.BlockHeader.ForDiff))
-	t.logger.With().Debug("Checking baseblock", base.Fields()...)
+	t.logger.With().Debug("checking baseblock", base.Fields()...)
 
-	lyr, ok := t.BlocksToBlocks[base.LayerIndex]
+	layerOpinions, ok := t.BlockOpinionsByLayer[base.LayerIndex]
 	if !ok {
 		return fmt.Errorf("baseblock layer not found %v, %v", block.BaseBlock, base.LayerIndex)
 	}
 
-	baseBlockOpinion, ok := lyr[base.ID()]
+	baseBlockOpinion, ok := layerOpinions[base.ID()]
 	if !ok {
 		return fmt.Errorf("baseblock not found in layer %v, %v", block.BaseBlock, base.LayerIndex)
 	}
 
-	// TODO: save and vote negative on blocks that exceed the max exception list size.
-	blockid := block.ID()
-
-	thisBlockOpinions := make(map[types.BlockID]vec)
+	// TODO: save and vote against blocks that exceed the max exception list size (DoS prevention)
+	opinion := make(map[types.BlockID]vec)
 	for blk, vote := range baseBlockOpinion.BlocksOpinion {
-		thisBlockOpinions[blk] = vote
+		opinion[blk] = vote
 	}
 
 	for _, b := range block.ForDiff {
-		thisBlockOpinions[b] = thisBlockOpinions[b].Add(support.Multiply(t.BlockWeight(blockid, b)))
+		opinion[b] = opinion[b].Add(support.Multiply(t.BlockWeight(block.ID(), b)))
 	}
 	for _, b := range block.AgainstDiff {
-		thisBlockOpinions[b] = thisBlockOpinions[b].Add(against.Multiply(t.BlockWeight(blockid, b)))
+		opinion[b] = opinion[b].Add(against.Multiply(t.BlockWeight(block.ID(), b)))
 	}
 
-	//TODO: neutral ?
-	t.logger.With().Debug("Adding block to blocks table", block.Fields()...)
-	t.BlocksToBlocks[block.LayerIndex][blockid] = Opinion{BlocksOpinion: thisBlockOpinions}
+	// TODO: neutral ?
+	t.logger.With().Debug("adding block to blocks table", block.Fields()...)
+	t.BlockOpinionsByLayer[block.LayerIndex][block.ID()] = Opinion{BlocksOpinion: opinion}
 	return nil
 }
 
 // HandleIncomingLayer processes all layer block votes
 // returns the old pbase and new pbase after taking into account the blocks votes
-func (t *turtle) HandleIncomingLayer(newlyr *types.Layer, inputVector []types.BlockID) (types.LayerID, types.LayerID) {
+// TODO: inputVector is unused here, fix this
+func (t *turtle) HandleIncomingLayer(newlyr *types.Layer, inputVector []types.BlockID) (pbaseOld, pbaseNew types.LayerID) {
```

Comment:
you removed `inputVector` from the signature everywhere else why not here as well ?